styles.css

```diff
@@ -0,0 +1,112 @@
+/* ****Start Mobile first**** */
+
+@media (max-width: 500px) {
+  .navbar_section {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-right: 40px;
+    padding-left: 20px;
+  }
+
+  .navbar_logo a {
+    font-size: 26px;
+    text-decoration: none;
+    color: rgb(41, 41, 106);
+    font-weight: bold;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+  }
+
+  .nav_botton {
+    cursor: pointer;
+  }
+
+  .dropdown_buttom {
+    position: relative;
+    display: inline;
+  }
+
+  .drop_downbtn {
+    display: none;
+    position: absolute;
+    background-color: white;
+    min-width: 160px;
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+    z-index: 1;
+  }
+
+  .drop_downbtn a {
+    color: black;
+    padding: 12px 12px;
+    text-decoration: none;
+    display: block;
+  }
+
+  .drop_downbtn a:hover {
+    background-color: rgb(69, 69, 183);
+  }
+
+  .dropdown_buttom:hover .drop_downbtn {
+    display: block;
+  }
+
+  .about_me {
+    background-image: url(./image/Header\ bg.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+
+    /* background-position: left; */
+    font-weight: 300;
+    padding: 10px;
+    color: rgb(41, 41, 106);
+    font-size: 25px;
+    padding-top: 100px;
+  }
+
```

Comment:
- [ ] Your background image is not fixed well since we are imitating the Figma design; kindly fix this so that those Icons on your background image are fixed as ones for Figma so that your website will look professional. I can see that you made some of the changes that were required in the previous review, but there is still something you still need to address.:wink:   don't forget also to put some style on your hamburger button :smile: 
your design :point_down: 
![improve1](https://user-images.githubusercontent.com/88894525/205050694-fe3d5bec-afa2-40e2-9513-1d94bce7103f.png)

Figma design :point_down: 
![Screenshot from 2022-12-01 14-06-30](https://user-images.githubusercontent.com/88894525/205050832-5c004539-5f96-45c3-8030-6343de5c8d36.png)